apps/search/index.js

```diff
@@ -0,0 +1,71 @@
+import express from 'express';
+import SearchBlocks from '../../collections/search_blocks';
+import { data as sd } from 'sharify';
```

Comment:
I don't think this is being used? You might want to turn on inline linting in your editor if possible. ESLint isn't running on CI yet.